app/assets/javascripts/discourse/app/lib/sidebar/admin-sidebar.js

```diff
@@ -230,9 +247,14 @@ export default class AdminSidebarPanel extends BaseCustomSidebarPanel {
 
     const navConfig = useAdminNavConfig(navMap);
```

Comment:
I think you should also allow us to define keywords inside `sidebar/admin-nav-map.js`, like how you added `query: { filter: "" }` there. On load you could add them all to the relevant sidebar link's key in `this.adminSidebarExperimentStateManager.keywords`